src/guide/backporting.md

```diff
@@ -7,7 +7,7 @@
 * [Skara documentation on backports](https://wiki.openjdk.org/display/SKARA/Backports)
 :::
 
-Development of the latest version of the JDK often results in bug fixes that might be interesting to include in some of the JDK update releases still being maintained. Moving a fix from a more recent release train (e.g. JDK 17) to an older release train (e.g. JDK 11) is called *backporting*.
+Development of the latest version of the JDK often results in bug fixes that might be interesting to include in some of the JDK update releases still being maintained. Moving a fix from a more recent release train (e.g. JDK 21) to an older release train (e.g. JDK 17) is called *backporting*.
```

Comment:
Would it make sense to use `N` and `N-4` instead of specific releases which must be continuously maintained?

Reply:
I think for the examples I'd prefer to use actual numbers. And I don't think they really needed to be updated, but I just did anyway since I was in the area.